octosql/config.go

```diff
@@ -0,0 +1,39 @@
+package octosql
+
+import (
+	"github.com/cube2222/octosql/config"
+	"github.com/cube2222/octosql/physical"
+	"os"
+)
+
+// Load configuration from the file using provided path
+func (e *OctosqlExecutor) LoadConfiguration(configPath string) error {
+
+	cfg := &config.Config{
+		DataSources: []config.DataSourceConfig{},
+		Execution:   map[string]interface{}{},
+	}
+
+	if _, err := os.Stat(configPath); err == nil {
+		cfg, err = config.ReadConfig(configPath)
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
```
_, err := os.Stat(configPath)
if err == nil {
 	cfg, err = config.ReadConfig(configPath)
 	if err != nil {
 		return err
 	}
 } else if os.IsNotExist(err) {
} else {
    // error other than not exist handling
}
```

Reply:
Oopsie, I forgot to change that. The error was discarded 😮 Thanks ❤️